<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CongratsWindow</class>
 <widget class="QDialog" name="CongratsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>383</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>472</width>
    <height>383</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>472</width>
    <height>383</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lableCheers">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Congratulations, you have solved a broblem!</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxHistory">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>451</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>History</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>113</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lableHistYourEff">
       <property name="text">
        <string>Your most effecient game:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lableHistComboMoves">
       <property name="text">
        <string>Number of moves:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lableHistComboTime">
       <property name="text">
        <string>Time: 00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lableHistYour">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Your best score:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lableHistBMoves">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>The least number of moves:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lableHistBTime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Your best time: 00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="buttonResetHistory">
    <property name="geometry">
     <rect>
      <x>359</x>
      <y>1</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>80</width>
      <height>10</height>
     </size>
    </property>
    <property name="text">
     <string>Reset history</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxScore">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Game score</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lableCurScore">
       <property name="text">
        <string>Your current score:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lableCurScoreMoves">
       <property name="text">
        <string>Number of moves: 0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignRight">
      <widget class="QLabel" name="lableCurScoreTime">
       <property name="text">
        <string>Time: 00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>451</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="buttonNewGame">
      <property name="text">
       <string>Play new game</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonTryAgain">
      <property name="text">
       <string>Try again (no score)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonQuit">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
